src/main/java/org/openrewrite/java/testing/mockito/MockitoJUnitAddMockitoSettingsLenientStrictness.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2021 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.testing.mockito;
+
+import org.jspecify.annotations.Nullable;
+import org.openrewrite.*;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.java.JavaTemplate;
+import org.openrewrite.java.RemoveAnnotation;
+import org.openrewrite.java.dependencies.DependencyInsight;
+import org.openrewrite.java.search.FindAnnotations;
+import org.openrewrite.java.search.UsesType;
+import org.openrewrite.java.tree.J;
+
+import java.util.Comparator;
+import java.util.Set;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class MockitoJUnitAddMockitoSettingsLenientStrictness extends ScanningRecipe<AtomicBoolean> {
+
+    private static String EXTEND_WITH_FQ = "org.junit.jupiter.api.extension.ExtendWith";
+    private static String MOCKITO_EXTENSION_FQ = "org.mockito.junit.jupiter.MockitoExtension";
+    private static String MOCKITO_SETTINGS_FQ = "org.mockito.junit.jupiter.MockitoSettings";
+    private static String MOCKITO_STRICTNESS_FQ = "org.mockito.quality.Strictness";
+
+    private static final String EXTEND_WITH_MOCKITO_EXTENSION = "@" + EXTEND_WITH_FQ + "(" + MOCKITO_EXTENSION_FQ + ".class)";
+
+    @Override
+    public String getDisplayName() {
+        return "Add `@MockitoSettings(strictness = Strictness.LENIENT)` when migration to JUnit 5";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Add `@MockitoSettings(strictness = Strictness.LENIENT)` when migration to JUnit 5.";
```

Comment:
It'd be good to describe _why_ we're making this change here, as it's likely to raise a few eye brows.